zotbins/Waste-Watcher-AWS-IoT-Core
# Changed serial monitor speed and added additional build flags

## Description

Changed serial monitor to the correct speed and added additional build flags.

## Type of change


## Testing and Verification Instructions

1. Compile the source code with the new changes.
2. Upload the application onto the ESP-32
3. Verify that messages can be read from the PlatformIO Serial monitor.

## Checklist:
